backend/app/Http/Controllers/Web/Admin/ProcessTemplateManagementController.php

```
@@ -156,6 +159,8 @@ public function show(ProductType $productType, ProcessTemplate $processTemplate)
                 'name' => $w->name,
                 'line_name' => $w->line?->name,
             ]),
+            // ISA-95 Equipment Classes (#52) for the step's workstation-type picker.
+            'workstationTypes' => \App\Models\WorkstationType::orderBy('name')->get(['id', 'name']),
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\WorkstationType::orderBy().


(staticMethod.notFound)